cmd/minikube/cmd/ssh.go

```diff
@@ -69,5 +73,5 @@ var sshCmd = &cobra.Command{
 }
 
 func init() {
-	sshCmd.Flags().Bool(nativeSSH, true, "Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.")
+	sshCmd.Flags().BoolVar(&nativeSSHClient, nativeSSH, true, "Use native Golang SSH client (default true). Set to 'false' to use the command line 'ssh' command when accessing the docker machine. Useful for the machine drivers when they will not start with 'Waiting for SSH'.")
```

Comment:
I am curious if we have other "bool" flags like this that could be fixed the same way ? (if not I am okay with merging this PR)

Reply:
No, this the only one.
Regards